.ddev/config.yaml

```
@@ -0,0 +1,256 @@
+name: ddev.com
+type: laravel
+docroot: dist
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
+additional_hostnames: []
+additional_fqdns: []
+mutagen_enabled: false
```

This is not a current config option. 

```suggestion
performance_mode: none
```